activity_calendar/static/fullcalendar/LICENSE.txt

```diff
@@ -0,0 +1,22 @@
+MIT License
```

Comment:
Expat License is okay and as we don't have a full JS pipeline yet I think it is fine to vendor this library. 

Can you add a link to the upstream source of this library, which version you have imported and which changes you have made in these files?